api/Navigator.json

```diff
@@ -978,8 +978,7 @@
               },
               {
                 "version_added": "21",
-                "prefix": "webkit",
-                "notes": "An outdated constraint syntax is still in use, but the syntax described here is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill."
```

Comment:
Another question for you @ddbeck. I've found that the different variants of `getUserMedia` take the same constraints dictionary since at least https://chromium.googlesource.com/chromium/src/+/8b986593441f4447834d7471440c6c1f644b0ed1, which is ~5 years ago. Is it OK to remove notes like this per https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#removal-of-irrelevant-features, or should I try to figure out when it changed and make sure that's reflected correctly for each browser? That seems like a lot of work and not particularly useful to anyone.

Reply:
This note is really bad to begin with and I fully support removing it. "the syntax described here" doesn't work out of the context of an MDN page and I'm 👎 on linking to polyfills. Make it go 🔥